types/rpc-handler.ts

```diff
@@ -111,11 +111,11 @@ export class RPCHandler implements HandlerInterface {
     return this._runtimeRpcs;
   }
 
-  public getNetworkId(): number {
+  public getNetworkId() {
     return this._networkId;
   }
 
-  public getNetworkName(): string {
+  public getNetworkName() {
```

Comment:
why remove  the type ?

Reply:
because the class values are typed there is no need to explicitly annotate a return type here as nothing is mutated in the function. Removal of `string` specifically is because it now uses the `NetworkName` union type.